src/test/pybind/test_rados.py

```diff
@@ -144,7 +144,10 @@ def test_ping_monitor(self):
         ret, buf, out = self.rados.mon_command(json.dumps(cmd), b'')
         for mon in json.loads(buf.decode('utf8'))['mons']:
             while True:
-                buf = json.loads(self.rados.ping_monitor(mon['name']))
+                output = self.rados.ping_monitor(mon['name'])
```

Comment:
in `rados.pyx`, if the returning length is 0, `None` is returned. so probably, we should instead check like:
```python
if output:
   continue
buf = json.loads(output)
```

or be more specific:
```python
if output is None:
   continue
buf = json.loads(output)
```